lib/elixir/test/elixir/kernel/sigils_test.exs

```diff
@@ -29,8 +31,22 @@ defmodule Kernel.SigilsTest do
     assert ~S(f\no) == "f\\no"
     assert ~S(foo\)) == "foo)"
     assert ~S[foo\]] == "foo]"
-    assert ~S(foo\
+  end
+
+  if windows?() do
+
```

Comment:
Can you remove the new lines after do, before else, after else and before end? Thank you!